src/multi_vector_simulator/D0_modelling_and_optimization.py

```diff
@@ -284,15 +288,39 @@ def simulating(dict_values, model, local_energy_system):
         -------
         Updated model with results, main results (flows, assets) and meta results (simulation)
         """
+        import warnings
```

Comment:
Shouldnt `warnings` be imported at the top of the module?

Reply:
yes you are right